collector/msft_fsrmquota.go

```diff
@@ -0,0 +1,232 @@
+package collector
+
+import (
+	//"strings"
+
+	"github.com/StackExchange/wmi"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/common/log"
+)
+
+func init() {
+	Factories["msft_fsrmquota"] = NewMSFT_FSRMQuotaCollector
+}
+
+// A MSFT_FSRMQuotaCollector is a Prometheus collector for WMI MSFT_FSRMQuota metrics
+type MSFT_FSRMQuotaCollector struct {
+	QuotasCount *prometheus.Desc
+	Path      *prometheus.Desc
+	PeakUsage *prometheus.Desc
+	Size      *prometheus.Desc
+	Usage     *prometheus.Desc
+	
+	
+	Description           *prometheus.Desc
+	Disabled              *prometheus.Desc
+	MatchesTemplate       *prometheus.Desc
+	SoftLimit             *prometheus.Desc
+	Template              *prometheus.Desc
+	Threshold             *prometheus.Desc
+	
+}
+
+// NewMSFT_FSRMQuotaCollector ...
+func NewMSFT_FSRMQuotaCollector() (Collector, error) {
+	const subsystem = "msft_fsrmquota"
+	return &MSFT_FSRMQuotaCollector{
+		QuotasCount: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "count"),
+			"Number of Quotas",
+			nil,
+			nil,
+		),
+		PeakUsage: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "peak_usage"),
+			"The highest amount of disk space usage charged to this quota. (PeakUsage)",
+			[]string{"quotaPath","quotaTemplate"},
+			nil,
+		),
+		Path: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "path"),
+			"A string that represents a valid local path to a folder. Must not exceed the value of MAX_PATH. Required. (Path)",
```

Comment:
We can skip the bits relating to how to create new quotas I think
```suggestion
			"A string that represents a valid local path to a folder. (Path)",
```